spec/07-variables.md

```diff
@@ -225,6 +291,83 @@ global variable. If that alias is passed to the `unset` intrinsic
 ([§§](10-expressions.md#unset)), only that alias is destroyed. The next time that function
 is called, a new alias is created with the current value of the global
 variable.
+<a name="undefined-global-variables"></a>
+
+**Undefined Global Variables**
+A distinction is made based on the context where an undefined global variable is used. 
+
+***byVal Context***
+
+PHP does not implicitly define an undefined global variable and uses `NULL` as substitution value instead. Furthermore, an `E_NOTICE`, stating that the corresponding variable was undefined, is emitted except for the following cases: 
+
+1.   The variable is used as single expression
+       1.  in a statement.
+       2.  as argument of [isset](10-expressions.md#isset).
+       3.  as argument of [empty](10-expressions.md#empty).
+       4.  as the left hand side of the [coalesce operator ??](10-expressions.md#coalesce-operator).
+
+Since undefined global variables are not defined implicitly, they stay undefined, a VSlot is not created for undefined variables used in a byValue context respectively.
```

Comment:
I did not want to imply it is unique to global variables. However, this claim is not made in the text and it is mainly your interpretation. It even states "a VSlot is not created for undefined variables used in a byValue context respectively" which is hold generall on purpose. It does not say "undefined global variables". But well, if it confuses you it might confuses others, so I am going to throw in a "in gerenal".